setup.py

```diff
@@ -83,6 +83,9 @@
     'scikit-learn==0.17.1',
     'ipykernel==4.4.1',
   ],
+  dependency_links=[
+     "https://storage.googleapis.com/cloud-ml/sdk/cloudml.latest.tar.gz",
```

Comment:
you can't add links to install_requires. Cloud ml is not published through pypi.